src/Illuminate/Support/Str.php

```diff
@@ -207,6 +198,17 @@ public static function is($pattern, $value)
         return false;
     }
 
+    /**
+     * Transliterate a UTF-8 value to ASCII.
```

Comment:
It looks like this sentence was copy and pasted from `Str::ascii()`'s docblock?

`ASCII::is_ascii()`'s comment is:

```
/**
 * Checks if a string is 7 bit ASCII.
 *
```

But Laravel docblock grammar for predicates is usually "Determine if...".

```
/**
 * Determine if a given string is 7 bit ASCII.
 *
```

Reply:
Ah, yes. Good catch! Yeh, I copied the whole function, but forgot to update the doc.